include/rdma/fi_prov.h

```diff
@@ -128,7 +131,6 @@ fi_param_get_bool(struct fi_provider *provider, const char *param_name, int *val
 	return fi_param_get(provider, param_name, value);
 }
 
-
```

Comment:
The change in this file was the subtraction of an empty line and copyright updates.  Seems unnecessary.